docs/02-app/02-api-reference/04-functions/server-actions.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: Server Actions
+nav_title: Server Actions
+description: API Reference for Next.js Server Actions.
+related:
+  title: Next Steps
+  description: For more information on what to do next, we recommend the following sections
+  links:
+    - app/building-your-application/data-fetching/forms-and-mutations
+---
+
+{/* TODO: This page will need to link back to React docs mentioned at the bottom */}
+
+Next.js integrates with React Actions to provide a built-in solution for [server mutations](/docs/app/building-your-application/data-fetching/forms-and-mutations).
+
+## Convention
+
+You can enable Server Actions in your Next.js project by enabling the **experimental** `serverActions` flag.
+
+```js filename="next.config.js"
+module.exports = {
+  experimental: {
+    serverActions: true,
+  },
+}
+```
+
+Server Actions can be defined in two places:
+
+- Inside the component that uses it (Server Components only)
+- In a separate file (Client and Server Components), for reusability. You can define multiple Server Actions in a single file.
+
+### With Server Components
+
+Create a Server Action by defining an asynchronous function with the `"use server"` directive at the top of the function body. This function should have **serializable arguments** and a **serializable return value**.
```

Comment:
On a separate note, it'll be good to clarify somewhere in our docs what these serializable values are. Seems like React has a different definition of what can be serialized: https://github.com/vercel/next.js/issues/54291

Reply:
Agreed – good chance to hit this in follow up.